app/helpers/dropdown_helper.rb

```diff
@@ -77,12 +77,18 @@ def allocated_to_options
   end
 
   def allocate_to_options
-    options = {
-      "Me" => current_user.uid,
-      "No one" => :no_one,
-    }
+    auditors = Audits::Plan
+                 .auditors(exclude: current_user)
+                 .sort_by(&:name)
+                 .unshift(OpenStruct.new(name: "Me", uid: current_user.uid))
```

Comment:
Do we want a test that these come up first in the list, or is that overkill?

Reply:
Addressed in 1ba9cdb